src/main/java/com/coreos/jetcd/internal/impl/LeaseImpl.java

```diff
@@ -313,13 +322,12 @@ private LeaseKeepAliveRequest newKeepAliveRequest(long leaseId) {
   }
 
 
-  private class KeepAliveListenerImpl implements KeepAliveListener {
+  private static class KeepAliveListenerImpl implements KeepAliveListener {
```

Comment:
Not always, sometime, especially with implementations, have a non static inner class is better as in can access instance variables.